src/pages/spawn/spawnHost/SpawnHostTableActions.test.tsx

```diff
@@ -12,7 +12,8 @@ const execCommand = jest.fn();
 const user = "bynn.lee";
 const hostUrl = "ec2-54-242-162-135.compute-1.amazonaws.com";
 
-test("CopySSHCommandButton", async () => {
+describe("CopySSHCommandButton", async () => {
+test("Tooltip text should change after clicking on the copy button", async () => {
```

Comment:
This also needs an accompanying bracket at the end of the the test

Reply:
done